<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>room</class>
 <widget class="QDialog" name="room">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1427</width>
    <height>672</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1427</width>
    <height>672</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1427</width>
    <height>672</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="out">
   <property name="geometry">
    <rect>
     <x>1310</x>
     <y>580</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
background-color: rgb(58, 58, 58);
	color: rgb(12, 162, 255);
font: 900 10pt &quot;Segoe UI&quot;;
border-radius:10px;
}
QPushButton:hover{
	background-color: rgb(255, 255, 255);
	color: rgb(34, 34, 34);
font: 900 10pt &quot;Segoe UI&quot;;
border-radius:10px;
}
QPushButton:pressed{
   
	background-color: rgb(37, 95, 255);
}</string>
   </property>
   <property name="text">
    <string>THOÁT</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>731</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="title">
    <string>Rooms Availability</string>
   </property>
   <widget class="QLabel" name="lbl101">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:7px;</string>
    </property>
    <property name="text">
     <string>101</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl102">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>102</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl103">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>103</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl104">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>104</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl105">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>70</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>105</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl204">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>260</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>204</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl205">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>260</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>205</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl203">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>203</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl201">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>201</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl202">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>202</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblinfo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>PHÒNG TRỐNG</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>430</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>PHÒNG ĐANG THUÊ</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>580</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
background-color: rgb(58, 58, 58);
	color: rgb(12, 162, 255);
font: 900 10pt &quot;Segoe UI&quot;;
border-radius:10px;
}
QPushButton:hover{
	background-color: rgb(255, 255, 255);
	color: rgb(34, 34, 34);
font: 900 10pt &quot;Segoe UI&quot;;
border-radius:10px;
}
QPushButton:pressed{
   
	background-color: rgb(37, 95, 255);
}</string>
   </property>
   <property name="text">
    <string>CHI TIẾT</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>360</y>
     <width>501</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>10</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(229, 1, 5);
font: 700 italic 28pt &quot;Roman&quot;;

</string>
   </property>
   <property name="text">
    <string>LUXURYOUS HOTEL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>580</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
background-color: rgb(58, 58, 58);
	color: rgb(12, 162, 255);
font: 900 10pt &quot;Segoe UI&quot;;
border-radius:10px;
}
QPushButton:hover{
	background-color: rgb(255, 255, 255);
	color: rgb(34, 34, 34);
font: 900 10pt &quot;Segoe UI&quot;;
border-radius:10px;
}
QPushButton:pressed{
   
	background-color: rgb(37, 95, 255);
}</string>
   </property>
   <property name="text">
    <string>CẬP NHẬT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>170</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 11pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>MÃ PHÒNG</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="room_no">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>170</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(30, 30, 30);
background-color: rgb(255, 255, 255);

font: 700 11pt &quot;Segoe UI&quot;;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>220</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 11pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>LOẠI PHÒNG</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="type">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>220</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(30, 30, 30);
background-color: rgb(255, 255, 255);

font: 700 11pt &quot;Segoe UI&quot;;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>280</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 11pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>CỠ PHÒNG</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="size">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>280</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(30, 30, 30);
background-color: rgb(255, 255, 255);

font: 700 11pt &quot;Segoe UI&quot;;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>330</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 11pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>GIÁ PHÒNG</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="price">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>330</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(30, 30, 30);
background-color: rgb(255, 255, 255);

font: 700 11pt &quot;Segoe UI&quot;;
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>580</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
background-color: rgb(58, 58, 58);
	color: rgb(12, 162, 255);
font: 900 10pt &quot;Segoe UI&quot;;
border-radius:10px;
}
QPushButton:hover{
	background-color: rgb(255, 255, 255);
	color: rgb(34, 34, 34);
font: 900 10pt &quot;Segoe UI&quot;;
border-radius:10px;
}
QPushButton:pressed{
   
	background-color: rgb(37, 95, 255);
}</string>
   </property>
   <property name="text">
    <string>THÊM</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>130</y>
     <width>521</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/New folder/company_card.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1431</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(62, 62, 62);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>781</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(58, 58, 58);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>80</y>
     <width>0</width>
     <height>571</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>0</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(104, 104, 104);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>140</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>ROOM INFO</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>100</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>101</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>102</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>103</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>104</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>105</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>201</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>202</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>203</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>204</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>205</string>
    </property>
   </item>
  </widget>
  <zorder>label_10</zorder>
  <zorder>label_9</zorder>
  <zorder>label_3</zorder>
  <zorder>label_8</zorder>
  <zorder>out</zorder>
  <zorder>groupBox</zorder>
  <zorder>pushButton</zorder>
  <zorder>tableView</zorder>
  <zorder>label_2</zorder>
  <zorder>update</zorder>
  <zorder>label_4</zorder>
  <zorder>room_no</zorder>
  <zorder>label_5</zorder>
  <zorder>type</zorder>
  <zorder>label_6</zorder>
  <zorder>size</zorder>
  <zorder>label_7</zorder>
  <zorder>price</zorder>
  <zorder>add</zorder>
  <zorder>label_11</zorder>
  <zorder>comboBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
